### PR TITLE
dependabot-triggers

### PR DESCRIPTION
**Pull request by cbardini**

### CODE REVIEW

**Review by jgaughan-aveva — APPROVED** [review]

**Review by anton-dahlin — APPROVED** [review]

File: azure-pipelines.yml [hunk 1]
```diff
@@ -30,21 +30,26 @@ resources:
...
   - name: containerDemandsADH
-    default: Agent.OS -equals Linux
+    type: object
+    default: ['Agent.OS -equals Linux]
+  - name: poolEDS
+    default: 00-OSIManaged-Build
   - name: containerDemandsEDS
-    default: PYTHON
+    type: object
+    default: ['PYTHON']
```
Comment (anton-dahlin) [inline azure-pipelines.yml:46]: There seems to be an issue with the changes related to the ['Python'] here.

From the pipeline 
image

**Review by anton-dahlin — CHANGES_REQUESTED** [review]
There seems to be an issue with the changes related to the ['Python'] here.

From the pipeline 
image

**Comment by anton-dahlin** [comment]
> There seems to be an issue with the changes related to the ['Python'] on line 48.
> 
> From the pipeline image

I found a quote that was missing, it runs now again

**Review by anton-dahlin — APPROVED** [review]

**Pull request merged**